source/incqueryserver-jupyter/iqs_jupyter/__init__.py

```diff
@@ -30,18 +30,15 @@
 from iqs_jupyter.core_extensions import *
 from iqs_jupyter.authentication import *
 
-if 'MmsRepositoryApi' in dir(iqs_client):
-    from iqs_jupyter.mms_extensions import *
-    from iqs_jupyter.mms_direct_extensions import *
+#if 'MmsRepositoryApi' in dir(iqs_client):
```

Comment:
I understand that we no longer ship trimmed down versions of the core API, but is there any specific need to remove these conditionals? I would expect them to work just fine.

Reply:
I found it inconsistent, because the new generator creates a different architecture... If we want it to stay, we may change it, but I thought it was easier for now to omit these features. This is a work in progress.

Reply:
Have we changed which openapi generator we use? Is there a specific reason why we instituted such a change? Does it otherwise generate better client code?

Reply:
Yes, the new version is generated using a gradle plugin, instead of a library.

Reply:
No, I mean OpenAPI (whether invoked from the CLI or via Gradle) has a selection of plugins called "generators". Are we still using the same one?

Reply:
@bergmanngabor - We would like to upgrade OpenApi generator to 5.x project-wise in IQS, I think it is reasonable to use newer version, because later we might face issues caused by the outdated generator.